Sources/Plasma/PubUtilLib/plPhysX/plSimulationMgr.cpp

```diff
@@ -295,7 +295,6 @@ bool plSimulationMgr::InitSimulation()
 {
     fSDK = NxCreatePhysicsSDK(NX_PHYSICS_SDK_VERSION, NULL, &gErrorStream);
     if (!fSDK) {
-        fLog->AddLine("Phailed to init PhysX SDK");
```

Comment:
This line was removed?

Reply:
`fLog == nullptr` and has been observed to cause crashes on MOULa when this section of the code was ported over.